### PR TITLE
[text-message] JSRS migration

### PR DESCRIPTION
**Pull request by bispodv**

### CODE REVIEW

**Comment by bispodv** [comment]
@kangawoo44 @abapayeViv Review, please

File: library-text-message-example/capsule.bxb [hunk 1]
```diff
@@ -12,6 +12,7 @@ capsule {
     section (Communication)
   }
   runtime-version (7) {
```
Comment (kangawoo44) [inline library-text-message-example/capsule.bxb:14]: let's update the runtime to the latest as well

**Comment by kangawoo44** [comment]
please state what's been tested as well after all the changes

**Comment by bispodv** [comment]
I have tested the following using a private revision:

Say "start" -> Touch the option _Tap to sign up for the  SMS mailing list_ -> Capsule answers "I ran into a problem"

This probably happens because my sample does not have a SIM card. I will get a SIM card so I'll be able to test this properly. I have extracted the logs from device but I was not able to find a clear indicative of what is going on. Note: the same thing happens for the non migrated capsule

**Comment by bispodv** [comment]
I will share the results as soon as I have tested

**Comment by kangawoo44** [comment]
Thanks for being thorough @bispodv !

**Comment by bispodv** [comment]
@kangawoo44 Looks like there is an issue when calling `viv.message` capsule.

image

I have updated `bixby.textMessage` lib version from 1.0.5 to 1.0.6 and the issue does not occur anymore, but when testing with a device using a private revision, after sending message it just answers "Here's what I found" instead of the success message "It's sent". I need to look the logs in Sumologic for the private revision, as soon as I get the access I requested

**Comment by kangawoo44** [comment]
what changed from 1.0.5 to 1.0.6 that resolved the issue?

**Comment by bispodv** [comment]
I couldn't figure out yet but that library calls the library "viv.message" and there is where the error occurs. Tried to figure out reading the notes but maybe it's better to look for the changes in the repo

image

Note that it is not related to the migration, there is only one simple JS file to be migrated

**Comment by kangawoo44** [comment]
oh, the library version. Ok if the issue is resolved we don't have to worry about it. Thanks for checking

**Comment by bispodv** [comment]
With the lib update I'm able to execute without any issue:

image

Only thing is that the warning for suggested lib version is showing as the recommended version is 1.0.5. Do you think that it would be a big deal to change the recommended version on Developer Console to 1.0.6?

image

**Comment by kangawoo44** [comment]
If that's the case, we'll just have to update the recommended version. I updated it.

**Comment by bispodv** [comment]
Thank you for updating, the "recommended version" deprecation is gone! But now there is only one last warning:

**Comment by kangawoo44** [comment]
ok this will have to wait, I'll get back to you on it when we finalize the decision on the deprecations coming from library capsules

**Review by kangawoo44 — APPROVED** [review]
since these warnings were there before the JSRS migration we'll merge this change and work on fixing the library warnings separately.

**Comment by bispodv** [comment]
@abapayeViv Hello. Would you be able to merge this PR? Thanks in advance

**Pull request merged**